src/qt/guiutil_font.h

```diff
@@ -9,15 +9,23 @@
 #include <QString>
 #include <QWidget>
 
+#include <array>
 #include <cstdint>
+#include <tuple>
```

Comment:
why `tuple` is here? 

Probably you need in `<utility>` if you want to have `std::pair`

```diff
-#include <tuple>
+#include <utility>
```

Reply:
As we've switched to `struct`, we can drop `<tuple>`